src/pages/index.js

```diff
@@ -44,12 +68,19 @@ function openProfilePopup() {
 
 function changeProfileInfo(event, data) {
   user.setUserInfo(data);
+  api.postUserInfo(data);
+  api.fetchUserInfo();
   editPopup.close();
 }
 
 function addLocationCard(event, data) {
   const newLocation = { name: data["title"], link: data["image-url"] };
-  cardSection.renderer(newLocation);
+  api
+    .postCard(newLocation)
+    .then((res) => res.json())
+    .then((data) => {
+      cardSection.renderer(data);
+    });
   event.target.reset();
   addFormValidator.disableSubmitButton();
```

Comment:
You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.